tangelo/problem_decomposition/dmet/_helpers/dmet_onerdm.py

```diff
@@ -34,16 +34,44 @@ def dmet_low_rdm(active_fock, number_active_electrons):
     """
 
     # Extract the occupied part of the one-particle RDM
-    num_occ = number_active_electrons / 2
+    num_occ = n_active_electrons / 2
```

Comment:
Should this be an integer ? The `/` performs floating point division, so if `n_active_electrons` is even you get a nice float that works well (ex: `3.0`) but otherwise you'd get something `5.5` or something.

Maybe we should use `//` or some rounding functions.